Create ui-c8y-1021-13-0-filtering-alarms-by-date-in-the-alarms-page-backport-7628.md (#2694)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-13-0-filtering-alarms-by-date-in-the-alarms-page-backport-7628.md

---
date: ""

> title: Filtering alarms by date in the alarms page BACKPORT (#7628)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Alarm filtering by date range in alarms page
```

> title: Filtering alarms by date in the alarms page BACKPORT (#7628)

Reviewer: ```suggestion
title: It is now possible to filter alarms by date in the alarms page.
```

> title: It is now possible to filter alarms by date in the alarms page.

Reviewer 2: based on the other changelog we pushed for filter by types i will propose this 
https://cumulocity.com/docs/change-logs/#ui-c8y-1021-0-0-adding-filtering-for-alarm-types-in-the-alarm-overview-6667

```suggestion
title: Date range filter added in the alarms overview 
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-60796
version: 1021.13.0
---

> Filtering alarms by date in the alarms page BACKPORT (#7628)

Author: The Claude AI suggested to use the following body:
```suggestion
To improve the usability of the alarms page, alarm filtering by a date range has been implemented. Users can now filter alarms by specifying a start and end date, allowing them to narrow down the displayed alarms to a specific time period. This change enhances the alarm management experience by providing more granular control over the alarms being viewed, making it easier for users to focus on relevant alarms within a desired timeframe.
```

> To improve the usability of the alarms page, alarm filtering by a date range has been implemented. Users can now filter alarms by specifying a start and end date, allowing them to narrow down the displayed alarms to a specific time period. This change enhances the alarm management experience by providing more granular control over the alarms being viewed, making it easier for users to focus on relevant alarms within a desired timeframe.

Reviewer 3: ```suggestion
To improve the usability of the **Alarms** page, alarm filtering by a date range has been implemented. Users can now filter alarms by specifying a start and end date, allowing to narrow down the displayed alarms to a specific time period. This change enhances the alarm management experience by providing more granular control over the alarms being viewed, making it easier for users to focus on relevant alarms within a desired timeframe.
```
